lib/galaxy/tools/deps/resolvers/conda.py

```diff
@@ -31,7 +31,7 @@
 
 DEFAULT_BASE_PATH_DIRECTORY = "_conda"
 DEFAULT_CONDARC_OVERRIDE = "_condarc"
-DEFAULT_ENSURE_CHANNELS = "conda-forge,r,bioconda,iuc"
+DEFAULT_ENSURE_CHANNELS = "iuc,bioconda,r,defaults,conda-forge"
```

Comment:
> We need to stick to the bioconda channel order

absolutely. but I don't think we do, given that for example for perl-gd the import tests pass (because the dependencies are pulled in from conda-forge). I don't completely understand the details yet, but to me it looks like galaxy is not respecting the channel order. Changing the order in planemo with `--conda_channels` has no effect whatsoever.